README.md

```diff
@@ -0,0 +1,25 @@
+# Clearbit Go Bindings
+
+
+## Usage
+
+
+```go
+package main
+
+import (
+  "fmt"
+  "github.com/clearbit/clearbit-go/clearbit"
+  "os"
+)
+
+func main() {
+  client := clearbit.NewClient(nil, os.Getenv("CLEARBIT_KEY"))
```

Comment:
@dcadenas let's improve the config - maybe take a ClearbitConfig object that let's you set the APIKey and defaults back to pulling out the key from ENV.